socs/db/suprsync.py

```diff
@@ -17,6 +19,40 @@
 Base = declarative_base()
 
 
+class TimecodeDir(Base):
```

Comment:
This docstring is great, please add an autoclass to the docs page under "Supporting APIs".